src/lib/debug.jsx

```diff
@@ -1,8 +1,9 @@
 import { useEffect } from 'react'
-import { useFlag } from 'cozy-flags'
+import useFlag from 'cozy-flags/dist/useFlag'
```

Comment:
Not a good look but could not get it imported any other way

Reply:
Est-ce qu'on pourrait pas regarder du côté de cozy-flag si on peut corriger le souci ?

Reply:
Possible oui mais cette MR est assez urgente car master est broken. @Crash-- que préfères-tu ?

Reply:
Créons l'issue sur le répo de cozy-flag et on traitera ça plus tard

Reply:
C'est vraiment important cette notion de si on a un problème suite à une maj de lib de faire l'issue sur la lib pour : 
- tracker le souci 
- pouvoir le corriger
- partager le fait qu'il y a un souci 

Garder ça dans son coin c'est pas productif pour cozy, car on tombera toujours sur le même souci et on reperdra encore et encore du temps sur le sujet

Reply:
issue created https://github.com/cozy/cozy-libs/issues/1330